#
#

Header header
int32 turnSignal    # -1 = left 0 = straight 1 = right
bool chassisLights  # 0 = off 1 = on
bool headLights     # 0 = off 1 = on
uint32 flagLight    # 0 = off 1 = white 2 = red 3 = blue
 